client/my-sites/earn/home.tsx

```diff
@@ -0,0 +1,252 @@
+/**
+ * External dependencies
+ */
+import { connect } from 'react-redux';
+import React, { FunctionComponent, Fragment } from 'react';
+import page from 'page';
+import { get, compact } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { useTranslate } from 'i18n-calypso';
+import { SiteSlug } from 'types';
+import { getSelectedSiteSlug } from 'state/ui/selectors';
+import getSiteBySlug from 'state/sites/selectors/get-site-by-slug';
+import { hasFeature, getCurrentPlan } from 'state/sites/plans/selectors';
+import { isJetpackSite } from 'state/sites/selectors';
+import { isRequestingWordAdsApprovalForSite } from 'state/wordads/approve/selectors';
+import PromoSection, { Props as PromoSectionProps } from 'components/promo-section';
+import QueryMembershipsSettings from 'components/data/query-memberships-settings';
+import QueryWordadsStatus from 'components/data/query-wordads-status';
+import {
+	FEATURE_WORDADS_INSTANT,
+	FEATURE_SIMPLE_PAYMENTS,
+	FEATURE_UPLOAD_PLUGINS,
+	PLAN_FREE,
+} from 'lib/plans/constants';
+
+interface ConnectedProps {
+	siteId: number;
+	selectedSiteSlug: SiteSlug;
+	isAJetpackSite: boolean;
+	isFreePlan: boolean;
+	hasSimplePayments: boolean;
+	hasWordAds: boolean;
+	hasUploadPlugins: boolean;
+	hasConnectedAccount: boolean;
+	hasSetupAds: boolean;
+}
+
+const Home: FunctionComponent< ConnectedProps > = ( {
+	siteId,
+	selectedSiteSlug,
+	isAJetpackSite,
+	isFreePlan,
+	hasSimplePayments,
+	hasWordAds,
+	hasUploadPlugins,
+	hasConnectedAccount,
+	hasSetupAds,
+} ) => {
+	const translate = useTranslate();
+
+	/**
+	 * Return the content to display in the Simple Payments card based on the current plan.
+	 *
+	 * @returns {object} Object with props to render a PromoCard.
+	 */
+	const getSimplePaymentsCard = () => {
+		const supportLink =
+			'https://en.support.wordpress.com/wordpress-editor/blocks/simple-payments-block/';
+		const cta = hasSimplePayments
+			? {
+					text: translate( 'Collect One-time Payments' ),
+					action: supportLink,
+			  }
+			: {
+					text: translate( 'Upgrade to Premium Plan' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'Upgrade to a Premium Plan!' )` *ES Score: 8*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).